workspace/welcome-to-xod/104-button/patch.xodp

```diff
@@ -1,7 +1,7 @@
 {
   "comments": [
     {
-      "content": "# Buttons\n\nHere is a `button` node from `xod/common-hardware`.\n\nTake a look at button outputs. It has a `PRS` pin, which is true whenever it is pressed. And `DONE` pin, which pulses when reading is done, wherever the button is pressed or not.\n\nThis is a standard idiom for side-effect nodes representing hardware or, for example, the Internet.\n\n## Exercise\n\nLet's make the LED glow while the button is pressed.\n\n1. Assemble the [circuit with a button and LED](https://xod.io/docs/tutorial/104-button/?utm_source=ide&utm_medium=ide_comment&utm_campaign=tutorial#circuit).\n2. Set the `PORT` value of the `button` node.\n3. Link `PRS` to `LUM`.\n4. Upload your patch.\n\nHold the button to turn the LED on.",
+      "content": "# Buttons\n\nHere is a `button` node from `xod/common-hardware`.\n\nTake a look at button outputs. It has a `PRS` pin, which is true whenever it is pressed. And `DONE` pin, which pulses when reading is done, wherever the button is pressed or not.\n\nThis is a standard idiom for side-effect nodes representing hardware or, for example, the Internet.\n\n## Exercise\n\nLet's make the LED glow while the button is pressed.\n\n1. Assemble the [circuit with a button and LED](https://xod.io/docs/tutorial/104-button/?utm_source=ide&utm_medium=ide_comment&utm_campaign=tutorial#circuit).\n2. Set the `PORT` value of the `button` node to `13` because it’s connected to the board port 13.\n3. Link `PRS` to `LUM`.\n4. Upload your patch.\n\nHold the button to turn the LED on.",
```

Comment:
```suggestion
      "content": "# Buttons\n\nHere is a `button` node from `xod/common-hardware`.\n\nTake a look at button outputs. It has a `PRS` pin, which is true whenever it is pressed. And `DONE` pin, which pulses when reading is done, wherever the button is pressed or not.\n\nThis is a standard idiom for side-effect nodes representing hardware or, for example, the Internet.\n\n## Exercise\n\nLet's make the LED glow while the button is pressed.\n\n1. Assemble the [circuit with a button and LED](https://xod.io/docs/tutorial/104-button/?utm_source=ide&utm_medium=ide_comment&utm_campaign=tutorial#circuit).\n2. Set the `PORT` value of the `button` node to `D13` because it’s connected to the board port 13.\n3. Link `PRS` to `LUM`.\n4. Upload your patch.\n\nHold the button to turn the LED on.",
```

Reply:
Or maybe even:

> ... to `D13` because it’s connected to the _digital_ board port # 13